org.gnome.Loupe.yml

```diff
@@ -0,0 +1,128 @@
+id: org.gnome.Loupe
+runtime: org.gnome.Platform
+runtime-version: '44'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+  - org.freedesktop.Sdk.Extension.llvm15
+command: loupe
+finish-args:
+  # Graphics
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  # Browse through photos
+  - --filesystem=host
```

Comment:
this is very broad as a permission.

Reply:
There hasn't been any progress on the relevant portal for years https://github.com/flatpak/xdg-desktop-portal/issues/463

People don't have images only in their Pictures folder. They also have graphics they are working on etc and want to flip through. Also, they want to open them from the file browser or terminal.